src/sagemaker/mxnet/defaults.py

```diff
@@ -12,4 +12,4 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
-MXNET_VERSION = '1.1'
+MXNET_VERSION = '1.2'
```

Comment:
Does this work? Should it be '1.2.1'?

Reply:
yeah, our images are tagged with both 1.2.1 and 1.2. We also did 0.12 for 0.12.1 and TF 1.4 for 1.4.1 so I'm following what we did before